dist/templates/welcome-screen.html

```diff
@@ -0,0 +1,122 @@
+<!--
```

Comment:
How much effort would it take to get rid of this? Module welcome screens are, to me, generally just an annoyance.

Reply:
No problem I remove it